### ISSUE TITLE
Comparison with None

### ISSUE THREAD

**Issue by chillenzer** [edbennett/python-testing-ci#20]
The idiomatic way of comparing with `None` is `x is (not) None` as opposed to `x == None`. I could see a weak argument about the symmetry in the 4 assertion statements in the edge and corner cases section to leave it that way but I would suggest using the idiomatic form.

**Comment by edbennett** [comment]
Agree, use `is None`. (Also adding extra spaces to line up equals signs isn't PEP8.)

**Comment by chillenzer** [comment]
The first change in my editor will have `black` applied on saving, so consider it fixed. =)
